cnd/src/db.rs

```diff
@@ -132,6 +133,25 @@ pub enum Error {
     SwapNotFound,
 }
 
+/// Data required to create a swap.
+///
+/// 'create' a swap is defined as the process of initiating a swap within `cnd`.
+/// The data required to do so is assumed to have been negotiated between the
+/// two parties prior to each creating the swap.
+#[derive(Debug, Clone)]
+pub struct CreatedSwap<A, B> {
+    /// Node specific swap identifier.
+    pub swap_id: LocalSwapId,
+    /// The protocol used on the alpha ledger.
```

Comment:
This also contains asset quantities, right?
Hence I am not sure `protocol` is the best description.
Maybe `parameters for the alpha ledger`?

Reply:
Re-named as suggested, thanks.